### PR TITLE
Document new functions for timestampz and bytea

### LINKED ISSUE #1033: Document new expressions for `timestampz` and `bytea` data types
### Related code PR

https://github.com/risingwavelabs/risingwave/pull/10462

### Which part(s) of the docs might be affected or should be updated? And how?

Document:
- timestamptz - timestamptz = interval under Date/time
- octet_length(bytea) = int32 (same as length(bytea)) under Bit string
- bit_length(bytea) = int32 under Bit string

### Reference

_No response_

### PR DESCRIPTION
**Pull request by CharlieSYH**
## Info

- **Description**

  - Document `timestamptz - timestamptz` under Date/time functions
  - Document `octet_length ` and `bit_length` under String functions
  - Add binary string functions.

- **Related code PR**

  https://github.com/risingwavelabs/risingwave/pull/10462

- **Related doc issue**
  
  Resolves https://github.com/risingwavelabs/risingwave-docs/issues/1033

## For reviewers

- **Preview**

  [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **upcoming** version of the documentation. ]

- **Key points**

  [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

File: docs/sql/functions-operators/sql-function-bitstring.md [hunk 1]
```diff
@@ -14,3 +14,5 @@ title: Bit string operators
 | &lt;&lt; | `operand1 << operand2` <br /> Bitwise left shift. | 1 &lt;&lt; 2 → 4 |
 | &gt;&gt; | `operand1 >> operand2` <br /> Bitwise right shift. | 4 &gt;&gt; 2 → 1 |
 
+
+The following string functions also support bit strings: `length`, `bit_length`, and `octet_length`. See [String functions](/docs/sql/functions-operators/sql-function-string.md#string-functions) for details.
```
Comment (xiangjinwu) [inline docs/sql/functions-operators/sql-function-bitstring.md]: Although the meaning of `octet_length` (and `bit_length`) is the same for `varchar` and `bytea`, the meaning of `length` varies:
* `length('🌊'::varchar) -> 1` is a shorthand for `char_length`, which counts the number of characters
* `length('🌊'::bytea) -> 4` is a shorthand for `octet_length`, which counts the number of bytes

Some related things:
* We are missing `char_length` and its longer equivalent `character_length`.
* The other operators on this page were treated as [bit string](https://www.postgresql.org/docs/current/functions-bitstring.html) incorrectly. We have not supported the bit string type yet. These are actually operators for smallint/int/bigint [here](https://www.postgresql.org/docs/current/functions-math.html#:~:text=integral_type%20%E2%86%92%20integral_type-,Bitwise%20AND,-91%20%26%2015%20%E2%86%92).
  * FYI the `length` for this unsupported bit string type is shorthand for `bit_length`.
* The page on `bytea` should be named [binary string](https://www.postgresql.org/docs/current/functions-binarystring.html).

File: docs/sql/functions-operators/sql-function-string.md [hunk 2]
```diff
@@ -28,6 +29,7 @@ title: String functions and operators
 |`lower` ( *input_string* ) → *output_string*|Converts the string to all lowercase.|`lower('TOM')` → `tom`|
 |`lpad`( *input_string*, *input_int* ) → *string* <br /><br /> lpad( *input_string*, *input_int*, *padding_string* ) → *string* | Pads the *input_string* on the left with spaces until it reaches the specified *input_int* length. If the *input_string* is longer than the *input_int* length, it is truncated to the specified length. Providing the optional *padding_string* replaces the spaces with the *padding_string*. | `lpad('42', 5)` → '&nbsp;&nbsp;&nbsp;42' <br /><br /> `lpad('42', 5, 'R')` → `RRR42` |
 |`ltrim` ( *input_string* [, *characters* ]) → *output_string*|Equals to `trim (LEADING)`.|`ltrim(' cake ')` → 'cake '<br/>`ltrim('abcxyzabc', 'cba')` → `xyzabc`|
+|`octet_length ( *input_string* )`|Returns number of bytes in the string, which is 8 times smaller than the `bit_length`. | `octet_length('wave')` → `4` |
```
Comment (xiangjinwu) [inline docs/sql/functions-operators/sql-function-string.md]: "8 times smaller than" sounds wired. Maybe omit this or say 1/8th.

File: docs/sql/functions-operators/sql-function-binarystring.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+---
+id: sql-function-binarystring
+slug: /sql-function-binarystring
+title: Binary string functions and operators
+---
+
+### `bit_length`
+
+Returns number of bits in the binary string, which is 8 times the [`octet_length`](#octet_length)).
```
Comment (xiangjinwu) [inline docs/sql/functions-operators/sql-function-binarystring.md]: * Redundant `)`
* Clicked the link in preview and it did not jump. Seems the hashtag is `octet_length-length`

**Review by xiangjinwu — APPROVED** [review]
Rest lgtm

**Review by emile-00 — APPROVED** [review]

**Pull request merged**